stripe-ui-core/build.gradle

```diff
@@ -67,6 +67,7 @@ dependencies {
     testImplementation testLibs.robolectric
     testImplementation testLibs.truth
     testImplementation testLibs.turbine
+    testImplementation project(':screenshot-testing')
```

Comment:
Nit: Prefer the project test implementation be above the library test implementations.